### PR TITLE
attune

### PR DESCRIPTION
**Pull request by untzag**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by untzag** [comment]
@ksunden could you please comment saying that you are OK being a maintainer of `attune`

**Comment by ksunden** [comment]
I am willing to be a maintainer

File: recipes/attune/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (jakirkham) [inline recipes/attune/meta.yaml]: Would update like so to [match the example]( https://github.com/conda-forge/staged-recipes/blob/fdc1ab5f16288dea426d5c53d4fccb7f37cd5c8a/recipes/example/meta.yaml#L33 ). These arguments are handled by conda-build behind the scenes.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment (untzag) [reply recipes/attune/meta.yaml]: thanks! should have read more carefully...

File: recipes/attune/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - setuptools
+    - pytest-runner
+  run:
+    - python >=3.5
+    - WrightTools >= 3.2.2
+    - numpy
```
Comment (scopatz) [inline recipes/attune/meta.yaml]: There should not be a space after the equals sogn

Comment (untzag) [reply recipes/attune/meta.yaml]: :+1:

**Pull request merged**